resources/i18n/en.json

```diff
@@ -218,7 +218,7 @@
       "EnvConfigWillDisappear": "Any unsaved environment variable and value will be disappeared.",
       "SharedMemoryAbbr": "SHMEM",
       "GPUSlot": "Slot",
-      "FolderAlias": "Alias",
+      "FolderAlias": "Alias(※ be careful about overridden of pre-existing folder)",
       "FolderAliasOverlapping": "Aliases overlap with other names that already exist",
```

Comment:
More than half of the words in the sentence are ellipsis. In light of the importance of the change, we need to fix this. (※ Please refer to the red box in the screenshot below:)

**screenshot**:
<img width="432" alt="alias-review" src="https://user-images.githubusercontent.com/46954439/149694631-deb3c7c7-6939-4183-af36-a3fa20a15017.png">

Reply:
It will be better to add question icon after "alias" to introduce how-to mount folder at any location.

Reply:
Reference: (?) icons on resource sliders. @SonMinWoo @lizable